Content.Client/_Starlight/Overlay/FlashImmunityCheck.cs

```diff
@@ -0,0 +1,69 @@
+
+
+using Content.Shared.Eye.Blinding.Components;
+using Content.Shared.Flash.Components;
+using Content.Shared.Inventory.Events;
+
+namespace Content.Client._Starlight.Overlay;
+
+public sealed class FlashImmunityCheck : EntitySystem
```

Comment:
call them by ss14 proper naming system.
```suggestion
public sealed class FlashImmunitySystem : EntitySystem
```